viite-UI/src/view/RoadAddressProjectEditForm.js

```diff
@@ -185,7 +198,29 @@
         '<footer>' + actionButtons() + '</footer>');
     };
 
-    var newRoadAddressInfo = function(){
+    var selectionForm = function(selection, selected, index){
+      return '<form id="roadAddressProjectForm_'+index+'" class="input-unit-combination form-group form-horizontal roadAddressProject">'+
+      '<label>Toimenpiteet,' + selection  + '</label>' +
+        ((selected.length == 2 && selected[0].linkId === selected[1].linkId) ?
+        '<span class="marker">'+markers[index]+'</span>'
+        : "") +
+      '<div class="input-unit-combination">' +
+      //TODO - Change the events in order to use id with numbering
```

Comment:
Is this TODO addressed, out of scope or already unneeded?

Reply:
Well noticed, it is already done